CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 0.5.1 - 2021-04-01
```

Comment:
@fiveisprime if you thought this would be an easy/small change, it is not - mostly because the last release of SWA was on some older versions of the shared packages. With that in mind, is it still worth a patch release? What's the timing of when you need this? We could also consider doing a full release in the next week or two rather than all this extra work for a patch release. It's hard to remember the state of the shared packages at that point in time and I almost feel more comfortable with what's on main.

Regardless, I think this date should be moved so that we can at least do a full test pass.

Reply:
There's no hard required date. Doing a full test pass makes sense to me. No big rush on this